components/sso-saml/org.wso2.carbon.identity.sso.saml.ui/src/main/java/org/wso2/carbon/identity/sso/saml/ui/client/SAMLSSOConfigServiceClient.java

```diff
@@ -126,4 +126,15 @@ public boolean removeServiceProvier(String issuerName) throws AxisFault {
         }
         return claimUris;
     }
+
+    public String[] getSigningAlgorithms() throws AxisFault{
+        String[] signingAlgorithms;
+        try {
+            signingAlgorithms = stub.getSigningAlgorithms();
+        } catch (Exception e) {
+            log.error("Error when reading signing algorithms",e);
+            throw new AxisFault(e.getMessage(),e);
```

Comment:
remove AxisFault and throw RemoteException, etc... why e.getMessage?
keep log and only throw message